hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/PhysicalRules.java

```diff
@@ -30,6 +31,7 @@ public static RuleSet getRuleSet() {
         return RuleSets.ofList(
                 // Filter rules
                 CalcPhysicalRule.INSTANCE,
+                CalcMergeRule.INSTANCE,
```

Comment:
1. the rule is in `opt.logical` package, is that correct?
2. why don't we use calcite's `CoreRules.CALC_MERGE`?

Reply:
1. I'll move it.
2. As Ozerov mentioned [here](https://github.com/hazelcast/hazelcast/issues/17058#issue-631615254), we needed to implement custom `Calc`-`Calc` rule, what was done.

Reply:
2. Still, I'd like t see:
  - why we need a different rule than the one calcite has
  - why we need it in the physical phase